src/crimson/mon/MonClient.h

```diff
@@ -81,6 +80,7 @@ class Client : public crimson::net::Dispatcher,
   MonSub sub;
 
 public:
+  MonMap monmap;
```

Comment:
why? i'd avoid exposing a member variable using `public`.

Reply:
Following [non-crimson MonClient](https://github.com/ceph/ceph/blob/master/src/mon/MonClient.h#L283), we need it when constructing `LogClient`

Reply:
i still prefer not exposing `monmap` as a public member variable.